spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -2053,6 +2061,126 @@ def setup_firefox_with_linux
         end
         it_behaves_like 'default case'
       end
+
+      context "id=collect_logs" do
+        context "server is not started" do
+          it "disables button with correct title" do
+            @record.status = "not responding"
+            error_msg = "Cannot collect current logs unless the Server is started"
+            expect(build_toolbar_disable_button("collect_logs")).to eq(error_msg)
+          end
+        end
+
+        context "server is started" do
+          context "log collecting is in progress, log_file.state = collection" do
+            it "disables button with correct title" do
```

Comment:
@jrafanie I got rid unneeded 'context' statements in all similar cases and I using just 'it'. This test cases are basically for testing toolbar button(whether is disabled or not) so I added '.. and disables button' as well.
Now it looks like:
```it 'log collection in progress with unfinished task and disables button'```